github/ipfs-examples.yml

```diff
@@ -13,7 +13,6 @@ members:
     - BigLep
     - hacdias
     - SgtPooki
-    - whizzzkid
```

Comment:
As removing the users is hard to revert, it is blocked by default in github as code. In other orgs, the users tend to be treated as alumni once all other permissions are removed from them. Let me know if you wanted to remove them or leave them in the org as an alumn.

Reply:
What is the effect of leaving them as alumni? I think its fine if left as alumni but he shouldnt have admin permissions on helia

Reply:
+1 to moving to alumni, we've been doing that for other folks post-nucleation